tests/test_hf_conversion_script.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2022 MosaicML LLM Foundry authors
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import sys
+
+from composer import Trainer
+
+from llmfoundry.models.mpt.modeling_mpt import ComposerMPTCausalLM
+
+# Add repo root to path so we can import scripts and test it
+repo_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+sys.path.append(repo_dir)
+import shutil
+from argparse import Namespace
+from typing import cast
+
+import pytest
+import torch
+import transformers
+from omegaconf import DictConfig
+from omegaconf import OmegaConf as om
+
+from scripts.inference.convert_composer_to_hf import main
+
+
+def delete_transformers_cache():
+    hf_cache_home = os.path.expanduser(
+        os.getenv(
+            'HF_HOME',
+            os.path.join(os.getenv('XDG_CACHE_HOME', '~/.cache'),
+                         'huggingface')))
+    HF_MODULES_CACHE = os.getenv('HF_MODULES_CACHE',
+                                 os.path.join(hf_cache_home, 'modules'))
+    if os.path.exists(HF_MODULES_CACHE) and os.path.isdir(HF_MODULES_CACHE):
+        shutil.rmtree(HF_MODULES_CACHE)
+
+
+def get_config(
+        conf_path='scripts/train/yamls/pretrain/testing.yaml') -> DictConfig:
+    os.environ['TOKENIZERS_PARALLELISM'] = 'false'
+    with open(conf_path) as f:
+        test_cfg = om.load(f)
+    return cast(DictConfig, test_cfg)
+
+
+def test_convert_and_generate_torch(tmp_path):
```

Comment:
should this have a pytest decorator?

Reply:
I think this is fine

I'll let @dakinggg answer (he wrote this part)

Reply:
yeah, this is ok. This is a CPU only test, the one below tests triton + gpu